pkg/policy/l4.go

```diff
@@ -233,8 +233,6 @@ func (l7 L7DataMap) addRulesForEndpoints(rules api.L7Rules, endpoints []CachedSe
 // the filter is left to be garbage collected!
```

Comment:
Please provide a more detailed commit message describing the motivation behind this change.

Reply:
Lost context for this, sorry.